src/index.js

```diff
@@ -51,9 +51,9 @@ export default class FacebookTokenStrategy extends OAuth2Strategy {
    * @param {Object} options
    */
   authenticate(req, options) {
-    let accessToken = (req.body && req.body[this._accessTokenField]) || (req.query && req.query[this._accessTokenField]) || (req.headers && req.headers[this._accessTokenField]);
+    let accessToken =  (FacebookTokenStrategy.parseAccessToken(req, this._accessTokenField, this._authorizationField));
```

Comment:
I'm not as familiar with ES6 so I'm not quite understanding how to implement your suggestion. The suggested change doesn't seem to handle the OAuth2 Authorization http header from the OAuth2 spec or parsing the Bearer token (which was what I was trying to add as part of this pull request)